dataplex/snippets/src/main/java/dataplex/SearchEntries.java

```diff
@@ -0,0 +1,85 @@
+ * limitations under the License.
+ */
+
+package dataplex;
+
+// [START dataplex_search_entries]
+import com.google.cloud.dataplex.v1.CatalogServiceClient;
+import com.google.cloud.dataplex.v1.Entry;
+import com.google.cloud.dataplex.v1.SearchEntriesRequest;
+import com.google.cloud.dataplex.v1.SearchEntriesResponse;
+import com.google.cloud.dataplex.v1.SearchEntriesResult;
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SearchEntries {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "MY_PROJECT_ID";
+    // How to write query for search: https://cloud.google.com/dataplex/docs/search-syntax
+    String query = "MY_QUERY";
+    // Maximum number of Entries that will be returned
+    int pageSize = 3;
+    // By default, keep page token empty
+    String pageToken = "";
+
+    SearchEntriesResponse searchEntriesResponse =
+        searchEntries(projectId, query, pageSize, pageToken);
+    List<Entry> entries =
+        searchEntriesResponse.getResultsList().stream()
+            // Extract Entries nested inside search results
+            .map(SearchEntriesResult::getDataplexEntry)
+            .collect(Collectors.toList());
+    entries.forEach(entry -> System.out.println("Entry name found in search: " + entry.getName()));
+    if (!searchEntriesResponse.getNextPageToken().isEmpty()) {
+      System.out.println(
+          "Page token of the next page: " + searchEntriesResponse.getNextPageToken());
+    }
+  }
+
+  // Method to search Entries located in projectId matching query, up to pageSize results, starting
+  // from pageToken. Returns SearchEntriesResponse that contains list of Entries and the token of
+  // the next page (if applicable)
+  public static SearchEntriesResponse searchEntries(
+      String projectId, String query, int pageSize, String pageToken) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (CatalogServiceClient client = CatalogServiceClient.create()) {
+      SearchEntriesRequest searchEntriesRequest =
+          SearchEntriesRequest.newBuilder()
+              .setPageSize(pageSize)
+              .setPageToken(pageToken)
+              // There are 2 ways to limit the scope of the search:
+              // 1) Set the "scope" field, with either "projects/<>" or "organizations/<>",
+              // which will limit the search to resources residing in given project / organization.
+              // 2) Leave "scope" field blank. In that case scope will be limited to organization in
+              // which project from "name" field is located.
+              // Please note that field "name" must always be filled, as the request itself will be
+              // attributed to that.
```

Comment:
such large comments in code samples point to a problem. either create two code samples to demonstrate different principles or remove this comment.
if you decide to use two code samples, please follow a code sample per class principle.